plugins/dll/EnactFrameworkRefPlugin.js

```diff
@@ -49,19 +49,13 @@ class EnactFrameworkRefPlugin {
 	constructor(options = {}) {
 		this.options = options;
 		this.options.name = this.options.name || 'enact_framework';
-		this.options.libraries = this.options.libraries || ['@enact', 'react', 'react-dom'];
+		this.options.libraries = this.options.libraries || ['@enact', 'react', 'react-dom', 'ilib'];
 		this.options.external = this.options.external || {};
 		this.options.external.publicPath =
 			this.options.publicPath || this.options.external.publicPath || this.options.external.path;
 
 		if (!process.env.ILIB_BASE_PATH) {
-			process.env.ILIB_BASE_PATH = path.join(
-				this.options.external.publicPath,
-				'node_modules',
-				'@enact',
-				'i18n',
-				'ilib'
-			);
+			process.env.ILIB_BASE_PATH = path.join(this.options.external.publicPath, 'node_modules', 'ilib');
```

Comment:
Should this have a fallback for enact 2.x ilib location?

Reply:
We could, though given the CLI PR adds aliases for `@enact/i18n/ilib` >> `ilib` for webpack/jest, in theory old locations are not expected. Currently the support is:
```
old cli + old enact + old app import references = success
new cli + old enact + old app import references + app-level ilib = success
new cli + new enact + old app import references + app-level ilib = success
new cli + new enact + new app import references + app-level ilib  = success
```
If we want to app additional logic to have new CLI build old enact (with old app import references and no app-level ilib), that's possible and can be done.